workitem/workitem_repository.go

```diff
@@ -87,6 +88,20 @@ func (r *GormWorkItemRepository) LoadFromDB(ctx context.Context, id uuid.UUID) (
 	return &res, nil
 }
 
+// LoadBatchFromDB returns the work items using IN query expression.
+func (r *GormWorkItemRepository) LoadBatchFromDB(ctx context.Context, ids []uuid.UUID) ([]WorkItemStorage, error) {
```

Comment:
do we really need to have this method that is only used by `LoadBatchByID` ?

Reply:
I was trying to mimic the pattern used by `LoadByID()` and `LoadFromDB()`

Reply:
As long as @aslakknutsen doesn't see fit for this method because of performance reasons, I'd say we should get rid of it and simply query each work item one by one.

Reply:
@kwk How querying work item one by one is helping compared to this approach?

Reply:
Thought behind having `Batch Load` is that it uses `IN` query and loads multiple items in a single query and I think that will be performant than loading multiple one by one.